src/Slider.js

```diff
@@ -5,18 +5,14 @@ import mdlUpgrade from './utils/mdlUpgrade';
 class Slider extends React.Component {
     static propTypes = {
         className: PropTypes.string,
-        max: PropTypes.number.isRequired,
-        min: PropTypes.number.isRequired,
-        onChange: PropTypes.func.isRequired,
+        max: PropTypes.number,
```

Comment:
Indeed. It seems they have a default min-max of 0-100 with 20 as the default value.
Agreed with disabled.